win32/ctags_vs2013.vcxproj.filters

```diff
@@ -120,6 +120,9 @@
     <ClCompile Include="..\main\portable-scandir.c">
       <Filter>Source Files\Main</Filter>
     </ClCompile>
+    <ClCompile Include="..\main\powershell.c">
+      <Filter>Source Files\Main</Filter>
+    </ClCompile>

```

Comment:
`powershell.c` is in the `parser` folder, not `main`.

Reply:
Thank you.

Doesn't test running at appveyor use win32/ctags_vs2013.vcxproj.filters ?

Ideally, this kind of mistake should be detected by a test.

Reply:
*.vcxproj.filters is only used by the Visual Studio IDE for showing the (virtual) directory tree.
It is not used for building, so it cannot detect by CI.